pkg/export/gce_token_source.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Kubernetes Authors.
+// Modifications copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package export
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+	"time"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/googleapi"
+	"k8s.io/client-go/util/flowcontrol"
```

Comment:
Kubernetes dependencies usually explode and basically always require Go modules hackery to get things to compile. The fewer distinct places we introduce them, the better the odds that we find a constellation that actually compiles.

Is this dependency needed or can we find a good stand-alone alternative?

Reply:
Removed this dependency by using the limiter from golang.org/x/time/rate instead